Makefile

```diff
@@ -25,7 +25,7 @@ endif
 
 ifndef NOCOLOR
   GREEN  := $(shell tput -Txterm setaf 2)
-  YELLOW := $(shell tput -Txterm setaf 3)
+  YELLOW := $(shell tput -Txterm seaf 3)
```

Comment:
was `setaf` changed in this line by mistake? I couldn't find anything regarding a `seaf` capname :sweat_smile:

Reply:
yes that a mistake and should be fixed - good catch!